spacy_curated_transformers/models/architectures.py

```diff
@@ -806,16 +807,35 @@ def build_pytorch_checkpoint_loader_v1(*, path: Path) -> Callable[
     path (Path):
         Path to the PyTorch checkpoint.
     """
+    return build_pytorch_checkpoint_loader_v2(path=path.parent)
```

Comment:
Think we can also update the v1 docstring to mention the support for SafeTensors.

Reply:
Added.